pkg/inits/config.go

```diff
@@ -27,7 +27,7 @@ func GetConfig() *entities.Config {
 	once.Do(func() {
 		viper.SetConfigName("config")
 		viper.SetConfigType("yaml")
-		viper.AddConfigPath("../../")
+		viper.AddConfigPath("/root")
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aakashgusain1990/productivite/pull/12#issuecomment-2424890211>#d2a798</a><span> - 10/20/2024, 05:44 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The hardcoded '/root' path may not be suitable for all environments and lacks flexibility.
</div>
<div id="fix">
<b>Fix:</b> Use an environment variable (CONFIG_PATH) to set the config path, with a fallback to "/root" if not set. This allows for flexibility across different environments while maintaining current behavior as a default.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -28,7 +28,12 @@
  		viper.SetConfigName("config")
  		viper.SetConfigType("yaml")
 -		viper.AddConfigPath("/root")
 +		configPath := os.Getenv("CONFIG_PATH")
 +		if configPath == "" {
 +			configPath = "/root"  // Default to /root if CONFIG_PATH is not set
 +		}
 +		viper.AddConfigPath(configPath)
 +		// Consider adding additional default paths like ".", "./config", etc.
 
  		if err := viper.ReadInConfig(); err != nil {
  			panic(err)
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged